.gitignore

```diff
@@ -1,3 +0,0 @@
-target
-Cargo.lock
-.DS_Store
```

Comment:
By the way, you deleted this file which was probably not intentional 😉

Reply:
added it back and yeah somehow it disappeared XD.

Reply:
Um, the file is still missing from the last commit. :)